docs/UserGuide.md

```diff
@@ -142,15 +210,24 @@ Allows you to add a vendor to WedLog, so that you can streamline coordination wi
 
 Format: `vendor add n/NAME [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]…`
 
->Tips:
-><br>
->- Items in square brackets are optional.
-><br>
->- A vendor can have any number of tags (including 0).
-><br>
->- A person can be recorded as both a guest and vendor.
->(e.g. Even if there is a guest named `Gia`, you are also able to add a vendor named `Gia`)
->- Refer to [Appendix A](#7-1-appendix-a-acceptable-values-for-parameters) for more details on the acceptable values for the parameters.
+<div class="alert alert-block alert-warning">
+
+**Parameter information:**
+
+* Parameters in square brackets are optional.
+* A vendor can have any number of tags (including 0).
+
+Refer to [Appendix A](#7-1-appendix-a-acceptable-values-for-parameters) for more details on the acceptable values for the parameters.
```

Comment:
This is a duplication of a bullet point under "Expected behavior upon failure" ... we should delete one of them.